lang/cs/Org.Apache.REEF.Client.Tests/YarnClientTests.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I'd be in favor of using `//` as it is easier to automate in the Visual Studio / ReSharper ecosystem. It really doesn't matter, though. Unless we ever want to to automate changes to it.